Logger/LogFactory.cs

```diff
@@ -2,10 +2,29 @@
 {
     public class LogFactory
     {
+        private string path;
```

Comment:
Fields should be named with underscore PascalCase
```suggestion
        private string _Path;
```